model/countries/AU/AU-model.yaml

```diff
@@ -0,0 +1,23 @@
+cut-off-children:  # don't expose nodes below these levels
+  - family-name
+
+cut-off-tokens:
+  - honorific-prefix
+  - honorific-suffix
+  - admin-area2
+  - admin-area3
+  - admin-area4
+  - locality2
+  - locality3
+  - locality4
+  - cross-streets
+  - building-name
+  - street-type
```

Comment:
I think you can drop these values. `cross-streets` won't be already present in the model due to the redefinition of the address alternative. `building-name` and `street-type` will just not be generated by our transpiler.

Reply:
I would recommend to follow the US style and have

```
cut-off-children:
  - street
  - family-name
```

instead of listing `street-type` here. Or do you have a specific reason why you added it?

Reply:
Adjusted the change!